### DISCUSSION TITLE
How to specify a different location for .env

### DISCUSSION THREAD

**Discussion by seunggs** [meltano/meltano#8330] [Q&A] [answered]
Hi, I'm running a project that has more than just meltano and I'd like meltano.yml to be able to refer to environment variables (i.e. from `.env` file outside of the meltano project root).

Is there a way to accomplish this?

Thanks in advance for all your help!

**Comment by WillDaSilva** [comment, marked as answer]
Hi @seunggs! Currently Meltano will only look for a `.env` file in the directory that contains the `meltano.yml` file, so I think the only way to accomplish what you're asking for currently would be to add a symbolic link named `.env` within the directory containing `meltano.yml` that points to the real `.env` file.

Please let me know if that works for you

**Reply by seunggs** [reply]
Hi @WillDaSilva thanks - I'll try that.

**Comment by edgarrmondragon** [comment]
FWIW there's also a proposal to add an `--env-file` option to the CLI: https://github.com/meltano/meltano/issues/3447

**Reply by seunggs** [reply]
Awesome - I've added my two cents there. Thanks!
